File: 2581.c
// Frime Number 2

#include <stdio.h>
#include <math.h>

int main(){
    int m, n;
    scanf("%d", &m);
    scanf("%d", &n);

    int sum=0, min=10001;
    for(int i = m; i <= n; i++){
        if(i == 1) continue;
        int flag = 1;
        for(int j = 2; j < (int)(sqrt((double)i)+1); j++){
            if(i%j==0){
                flag = 0;
                break;
            }
        }
        if(flag){
            sum += i;
            min = (min > i) ? i : min;
        }
    }
    if(sum == 0) printf("-1");
    else printf("%d\n%d", sum, min);
    return 0;
}

File: 2581.py
# Prime Number 2
"""
m,n=int(input()),int(input())
#l=list(all(i%j for j in range(2,int(i**.5)+1))*i>1 for i in range(m,n+1))
l2=[i for i in range(m,n+1) if all(i%j for j in range(2,int(i**.5)+1))*i>1]
print(f"{sum(l2)}\n{min(l2)}" if l2!=[] else -1)"""

#l=[i for i in range(int(input()),int(input())+1)if all(i%j for j in range(2,int(i**.5)+1))*i>1];print(-(l==[])or f"{sum(l)}\n{min(l)}")
print(*map(int,input().split()))
